Dockerfile

```diff
@@ -77,7 +77,7 @@ RUN mkdir /btfs /btns \
 # Expose the fs-repo as a volume.
 # start_btfs initializes an fs-repo if none is mounted.
 # Important this happens after the USER directive so permission are correct.
-VOLUME $IPFS_PATH
+VOLUME $BTFS_PATH
 
 # The default logging level
 ENV IPFS_LOGGING ""
```

Comment:
Do we want to change this "IPFS_LOGGING" as well?

Reply:
@taiyangc Should we change "IPFS_LOGGING" as well?

Reply:
It probably should, but not just here, the logging code needs to change `IPFS_LOGGING` -> `BTFS_LOGGING` as well.

Reply:
Okay, I will merge this PR and then make a separate PR to change the logging nomenclature.